gulpfile.js

```diff
@@ -10,21 +10,22 @@
  */
 'use strict';
 
-var gulp = require('gulp');
+var autoprefixer = require('gulp-autoprefixer');
 var bulk = require('gulp-sass-bulk-import');
-var sass = require('gulp-sass');
 var civicrmScssRoot = require('civicrm-scssroot')();
+var cssmin = require('gulp-cssmin');
+var gulp = require('gulp');
+var karma = require('karma');
+var path = require('path');
 var postcss = require('gulp-postcss');
-var postcssPrefix = require('postcss-prefix-selector');
 var postcssDiscardDuplicates = require('postcss-discard-duplicates');
-var stripCssComments = require('gulp-strip-css-comments');
-var transformSelectors = require('gulp-transform-selectors');
-var cssmin = require('gulp-cssmin');
+var postcssPrefix = require('postcss-prefix-selector');
 var rename = require('gulp-rename');
+var sass = require('gulp-sass');
+var stripCssComments = require('gulp-strip-css-comments');
 var sourcemaps = require('gulp-sourcemaps');
-var autoprefixer = require('gulp-autoprefixer');
-var karma = require('karma');
-var path = require('path');
+var transformSelectors = require('gulp-transform-selectors');
+var runBackstopJS = require('./gulp-tasks/backstopjs.js');
```

Comment:
Lets call this, something like `backstopjsTask`. Then we can follow `<>-task` syntax in future.